src/components/AvatarGroup/AvatarGroup.module.scss

```diff
@@ -0,0 +1,66 @@
+@import "src/components/Avatar/Avatar.scss";
+@import "../../styles/themes";
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  position: relative;
+}
+
+.avatarContainer {
+  margin-left: -8px;
```

Comment:
I think we have now variables for negative spacing. can you please check it up with Yonatan ben ari?

Reply:
We don't have them, Yonatan told me that you and Orr had some concerns about it
Does this look ok `margin-left: calc(-1 * var(--spacing-small));` ?